lib/tbd/psi.rb

```diff
@@ -1815,40 +1815,39 @@ def process(model = nil, argh = {})
 
     # Unless a user has set the thermal bridge type of an individual edge via
     # JSON input, reset any subsurface's head, sill or jamb edges as (mild)
-    # transitions in close proximity with other similar subsurface edges.
-    # User-set tolerance must be greater than default TBD tolerance (0.01 m).
-    # Both edges' origin and terminal vertices must be in close proximity. Edges
+    # transitions when in close proximity to another subsurface edge. Both
+    # edges' origin and terminal vertices must be in close proximity. Edges
     # of unhinged subsurfaces are ignored.
-    if argh[:sub_tol] > TOL
-      edges.each do |id, edge|
-        nb    = 0                            # linked subsurfaces (i.e. "holes")
-        match = false
-        next if edge.key?(:io_type)          # skip if set in JSON
-        next unless edge.key?(:v0)
-        next unless edge.key?(:v1)
-        next unless edge.key?(:psi)
-        next unless edge.key?(:surfaces)
-
-        edge[:surfaces].keys.each do |identifier|
-          next unless holes.key?(identifier)
+    edges.each do |id, edge|
+      nb    = 0                            # linked subsurfaces (i.e. "holes")
+      match = false
+      next if edge.key?(:io_type)          # skip if set in JSON
+      next unless edge.key?(:v0)
+      next unless edge.key?(:v1)
+      next unless edge.key?(:psi)
+      next unless edge.key?(:surfaces)
 
-          if holes[identifier].attributes.key?(:unhinged)
-            nb = 0 if holes[identifier].attributes[:unhinged]
-            break  if holes[identifier].attributes[:unhinged]
-          end
+      edge[:surfaces].keys.each do |identifier|
+        break    if match
+        next unless holes.key?(identifier)
 
-          nb += 1
+        if holes[identifier].attributes.key?(:unhinged)
+          nb = 0 if holes[identifier].attributes[:unhinged]
+          break  if holes[identifier].attributes[:unhinged]
         end
 
-        next unless nb == 1   # only process edge if linking a single subsurface
+        nb += 1
+        match = true if nb > 1
```

Comment:
Fixing a bug from previous commits. Previous change prevented catching common _transition_ edges shared between 2x subsurfaces (regardless of `:sub_tol` value).